src/webviews/apps/plus/graph/graph.tsx

```diff
@@ -91,11 +94,40 @@ export class GraphApp extends App<State> {
 				});
 				break;
 
+			case DidChangeConfigurationNotificationType.method:
+				this.log(`${this.appName}.onMessageReceived(${msg.id}): name=${msg.method}`);
+
+				onIpc(DidChangeConfigurationNotificationType, msg, params => {
+					this.setState({ ...this.state, mixedColumnColors: this.getGraphColors(params.config.graph) });
+					this.refresh(this.state);
+				});
+				break;
+
 			default:
 				super.onMessageReceived?.(e);
 		}
 	}
 
+	private getGraphColors(config: GraphConfig | undefined): CssVariables {
+		// this will be called on theme updated as well as on config updated since it is dependent on the column colors from config changes and the background color from the theme
+		const body = document.body;
+    	const computedStyle = window.getComputedStyle(body);
+		const bgColor = computedStyle.getPropertyValue('--color-background');
+		const columnColors = ((config?.columnColors) != null) ? config.columnColors : ['#00bcd4', '#ff9800', '#9c27b0', '#2196f3', '#009688', '#ffeb3b', '#ff5722', '#795548'];
```

Comment:
It's ok to load colors from the config but we are not setting the all the `cssVariables` needed to paint all edge columns. See my comment: https://github.com/gitkraken/vscode-gitlens/pull/2105#discussion_r926643618

Now, if we change one color from the settings, only the color of the branch is changed and not the entire edge column:
https://user-images.githubusercontent.com/90025366/180764253-0bbe6bc3-d432-4c79-822e-e4a5348141fe.mov

Reply:
@miggy-e  in `base.jsonc` (GKC side) and `columnColors.less` (library side) files we use also `--graph-color-0-f10`, `--graph-color-0-f50`, `--graph-color-1-f10`, etc. 

For example in base.jsonc we have:
```
// graph color variations
"graph-color-5-f10": "fade(@graph-color-5, 10%)",
"graph-color-6-f10": "fade(@graph-color-6, 10%)",
"graph-color-7-f10": "fade(@graph-color-7, 10%)",
"graph-color-8-f10": "fade(@graph-color-8, 10%)",
"graph-color-9-f10": "fade(@graph-color-9, 10%)",
"graph-color-0-f10": "fade(@graph-color-0, 10%)",
"graph-color-1-f10": "fade(@graph-color-1, 10%)",
"graph-color-2-f10": "fade(@graph-color-2, 10%)",
"graph-color-3-f10": "fade(@graph-color-3, 10%)",
"graph-color-4-f10": "fade(@graph-color-4, 10%)",
"graph-color-5-f50": "fade(@graph-color-5, 50%)",
"graph-color-6-f50": "fade(@graph-color-6, 50%)",
"graph-color-7-f50": "fade(@graph-color-7, 50%)",
"graph-color-8-f50": "fade(@graph-color-8, 50%)",
"graph-color-9-f50": "fade(@graph-color-9, 50%)",
"graph-color-0-f50": "fade(@graph-color-0, 50%)",
"graph-color-1-f50": "fade(@graph-color-1, 50%)",
"graph-color-2-f50": "fade(@graph-color-2, 50%)",
"graph-color-3-f50": "fade(@graph-color-3, 50%)",
"graph-color-4-f50": "fade(@graph-color-4, 50%)",
...
```

Do you think we should include them too or it is not necessary?

Reply:
I added them in line 124

Reply:
Ok, I have not seen it.